common/span.h

```diff
@@ -280,9 +278,7 @@ class SpanBase : public SafeBool<Derived<ValueType> > {
 	inline size_type byteSize() const { return impl().size() * sizeof(value_type); }
 
 #if !defined(_MSC_VER)
```

Comment:
This may be interesting to see if these are needed as well.

Reply:
Related commit 4564b84d918e13bfceed4338aa3111bf468b0f0e mentions Visual Studio 2015. I think we still support it?

Reply:
OK. I am not sure why this error is here but I didn't study the code enough.